Detectors/TPC/workflow/src/tpc-reco-workflow.cxx

```diff
@@ -146,18 +149,23 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   }
 
   bool doMC = not cfgc.options().get<bool>("disable-mc");
-  return o2::tpc::reco_workflow::getWorkflow(&gPolicyData,                                      //
-                                             tpcSectors,                                        // sector configuration
-                                             gTpcSectorMask,                                    // same as bitmask
-                                             laneConfiguration,                                 // lane configuration
-                                             doMC,                                              //
-                                             nLanes,                                            //
-                                             inputType,                                         //
-                                             cfgc.options().get<std::string>("output-type"),    //
-                                             !cfgc.options().get<bool>("no-ca-clusterer"),      //
-                                             !cfgc.options().get<bool>("no-tpc-zs-on-the-fly"), //
-                                             cfgc.options().get<bool>("zs-10bit"),              //
-                                             cfgc.options().get<float>("zs-threshold"),         //
-                                             !cfgc.options().get<bool>("ignore-dist-stf")       //
+  auto wf = o2::tpc::reco_workflow::getWorkflow(&gPolicyData,                                      //
+                                                tpcSectors,                                        // sector configuration
+                                                gTpcSectorMask,                                    // same as bitmask
+                                                laneConfiguration,                                 // lane configuration
+                                                doMC,                                              //
+                                                nLanes,                                            //
+                                                inputType,                                         //
+                                                cfgc.options().get<std::string>("output-type"),    //
+                                                !cfgc.options().get<bool>("no-ca-clusterer"),      //
+                                                !cfgc.options().get<bool>("no-tpc-zs-on-the-fly"), //
+                                                cfgc.options().get<bool>("zs-10bit"),              //
+                                                cfgc.options().get<float>("zs-threshold"),         //
+                                                !cfgc.options().get<bool>("ignore-dist-stf")       //
   );
+
+  // write the configuration used for the digitizer workflow
+  o2::conf::ConfigurableParam::writeINI("o2itsrecoflow_configuration.ini");
```

Comment:
Hi @shahor02 : This seems to be a copy and paste error here. For the comment, this is not the digitizer worklow but TPC, and the options are store in the ini for ITS not TPC.